docs/serving/getting-started-knative-app.md

```diff
@@ -72,31 +88,45 @@ spec:
 ```
 
 If you want to deploy the sample app, leave the config file as-is. If you're
-deploying an image of your own app, update the name of the app and the URL of
+deploying an image of your app, update the name of the app and the URL of
 the image accordingly.
 
-## Deploying your app
-
 From the directory where the new `service.yaml` file was created, apply the
 configuration:
 
 ```bash
 kubectl apply --filename service.yaml
 ```
 
-Now that your service is created, Knative will perform the following steps:
-
-- Create a new immutable revision for this version of the app.
-- Perform network programming to create a route, ingress, service, and load
-  balancer for your app.
-- Automatically scale your pods up and down based on traffic, including to zero
-  active pods.
-
-### Interacting with your app
+## Interacting with your app
 
 To see if your app has been deployed successfully, you need the URL created by Knative.
 
 1. To find the URL for your service, enter:
+   ```shell
```

Comment:
I think tabs would make this part look a lot nicer.

Reply:
Let me try that. Not sure how it looks like when rendered as it is complex content.

Reply:
I've kicked off a preview. It's apparently triggered by `/lgtm` or `approve` labels, so I'm putting a `/hold` in, too.